bin/genesis

```diff
@@ -3096,15 +3096,6 @@ ci_update() {
 	fi
 }
 
-ci_pipeline_yaml() {
```

Comment:
kudos on picking the right one to remove.  the remaining definition is actually correct 🤣